src/main/java/uk/ac/ebi/tsc/tesk/service/TesService.java

```diff
@@ -34,7 +34,7 @@
      * @param view   - one of {@link TaskView} values, decides on how much detail is put in results
      * @return - TES task details
      */
-    @PostAuthorize("(#user.username == returnObject.logs[0].metadata['USER_ID'] AND returnObject.logs[0].metadata['GROUP_NAME'] != null AND #user.isGroupMember(returnObject.logs[0].metadata['GROUP_NAME']))" +
+    @PostAuthorize("(authentication.authenticated AND @authorisationProperties.isIgnoreGroupMembership() AND #user.username == returnObject.logs[0].metadata['USER_ID']) OR (#user.username == returnObject.logs[0].metadata['USER_ID'] AND returnObject.logs[0].metadata['GROUP_NAME'] != null AND #user.isGroupMember(returnObject.logs[0].metadata['GROUP_NAME']))" +
             " OR (#user.teskAdmin)" +
```

Comment:
This condition can be simplified: isOwner and either ignoreMembership or belong to a group.